vignettes/GenomicFiles.Rmd

```diff
@@ -0,0 +1,703 @@
+---
+editor_options: 
+  markdown: 
+    wrap: 72
+---
+
+---
+title: "GenomicFiles"
+author: Valerie Obenchain,Michael Love,Lori Shepherd,Martin Morgan
+date: March 04, 2023
+package: GenomicFiles
+vignette: >
+  %\VignetteIndexEntry{GenomicFiles}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+output:
+  BiocStyle::html_document
+  
+---
```

Comment:
- [ ] Removing lines 1-6 because we don't need to wrap lines at 72 characters. There should be one YAML header.
- [ ] The title of this file should be "Introduction to Genomic Files". This is what you see in https://bioconductor.org/packages/3.17/bioc/vignettes/GenomicFiles/inst/doc/GenomicFiles.pdf.
- [ ] List the authors as follows
  ```
  author:
    - Valerie Obenchain
    - Michael Love
    - Lori Shepherd
    - Martin Morgan
  ```
- [ ] You can add yourself as a contributor here with
  ```
    - name: Your Name
      affiliation: "Vignette translation from Sweave to R Markdown / HTML"
   ```
  Be careful with the spacing.
- [ ] Rather than hard code the date, use the following code so that it's always "current" when the file is regenerated:
  ```
  date: "`r format(Sys.time(), '%B %d, %Y')`"
  ```
- [ ] On line 13, the VignetteIndexEntry should be `Introduction to GenomicFiles`. It's typically the same as the title.
- [ ] Remove the extra space between `BiocStyle: html_document` and the `---`.